.evg.yml

```diff
@@ -265,15 +265,16 @@ tasks:
         params:
           shell: "bash"
           script: |
-            set -v
```

Comment:
⚠️  ... this was printing out details like the segment write key in the evergreen logs